src/_posts/2019-10-16.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Lightning Hacks 3: O tempo acabou"
+layout: post
+---
+
+# Lightning Hacks 3: O tempo acabou
+> Dia 18/9/2019, MC: Andrew Ijano
+
+Nesse Lightning Hacks, _todos_ os apresentadores tiveram seu tempo limitado pelo relógio.
+Ou seja, no final de toda apresentação, todos os hackers falaram "Ah, meu tempo acabou...".
+Foi uma reunião extremamente dinâmica e intensa!
+
+<hr>
+
+## 1. A beleza de códigos nojentos (feat. Union Find)
+> Hacker: Teos (Pedro Sousa)
+
+Teos falou sobre a estrutura Union Find, introduzindo as eurísticas de path compression e union by weight.
+Uma rápida implementação no final da apresentação ajudou a acrescentar uma camada adicional aos códigos
+da maratona: eles não são apenas feios, são bonitos também!
+
+## 2. Se sentindo como Von Neumann: brincando com somatórios como um computeiro
+> Hacker: Bento Pereira
+
+Anos atrás, em tempos imemoriais, o matemático John Von Neumann foi apresentado com um problema 
+[sobre uma pobre abelha](https://courses.cs.vt.edu/~cs1104/ProblemSolving/Trains/Train.html). 
+O bater das pequenas asas dessa abelha fez com que, por meio do conhecido _efeito abelha infinita_, 
+um jovem se sentisse compelido a mostrar como podemos resolver somatórios aparentemente simples em 
+segundos usando algo natural para computeiros: os números binários.
+
+## 3. Uma Perspectiva geométrica à acordes
+> Hacker: P2 (Pedro Pereira)
+
+Foi apresentada uma aplicação super simples: <https://chromatic.razgrizone.now.sh/>.
+Com ela, Pedro mostrou o que é uma escala e um acorde através de um polígono em um círculo das 12 notas.
+Conclusões incríveis foram tiradas: um acorde está dentro de uma escala se as diagonais que o acorde forma
+estão todas dentro do polígono!
+
+O código fonte da aplicação [pode ser encontrado aqui.](https://github.com/pedro823/chromatic)
+
+## 4. Funtores e Applicatives em Haskell
+> Hacker: spoonm
+
+Uma tentativa de explicar em 5 minutos dois padrões recorrentes em programação funcional que foram 
+tirados de Teoria de Categorias: funtores e applicatives (funtores monoidais que preservam força tensorial), 
```

Comment:
This post shouldn't be here! Please remove it.

Reply:
Hm, it seems that I don't know how to update the files after a git checkout... Anyway, post removed